src/Esprima/JavascriptParser.cs

```diff
@@ -1192,7 +1192,7 @@ private INode ParseGroupExpression()
                                 if (expr.Type == Nodes.SequenceExpression)
                                 {
                                     var sequenceExpression = expr.As<SequenceExpression>();
-                                    var reinterpretedExpressions = new List<Expression>();
+                                    var reinterpretedExpressions = new Ast.List<Expression>();
```

Comment:
I did notice couple of places where the list size is known beforehand like here, so it could be supplied to constructor. But it's an existing problem for sure.